grassroots-backend/src/grassroots-shared/Contact.entity.dto.ts

```diff
@@ -21,6 +21,10 @@ export class CreateContactInDto {
   phoneNumber!: string;
 }
 
+export class CreateContactInDtoArray {
```

Comment:
My only question is on the use of `Array` in our naming.

Rather than a database `In` class, this seems more like a request object that has an array of contacts on it, but also might have other things in the future.

Reply:
Yup, good call.

WDYT about CreateBulkContactInDto? (or RequestDTO).

Reply:
`CreateBulkContactRequestDTO` feels good to me. I wonder if `BulkCreateOut` should become `CreateBulkContactResponseDTO`.

Reply:
My thought was that BulkCreateOut might be agnostic to the type of the object bulk created, but in retrospect, that's probably silly. 

SGTM.